spec/models/chargeback_vm_spec.rb

```diff
@@ -67,7 +67,7 @@ def set_tier_param_for(metric, param, value, num_of_tier = 0)
     temp = {:cb_rate => chargeback_rate, :tag => [c, "vm"]}
     ChargebackRate.set_assignments(:compute, [temp])
 
-    Timecop.travel(Time.parse('2012-09-01 23:59:59').utc)
+    Timecop.travel(Time.parse('2012-09-01 23:59:59Z'))
```

Comment:
@gtanzillo I wonder if adding the extra `.utc` would be OK and keep rubocop happy!